staging/src/slime.io/slime/modules/meshregistry/pkg/source/zookeeper/conversion.go

```diff
@@ -63,43 +47,6 @@ func (e Error) Error() string {
 	return e.msg
 }
 
-func getMetaServicePort(payload ServiceInstancePayload) *networking.Port {
```

Comment:
Doing code changes that have nothing to do with the main idea of the PR will make the review more difficult

Reply:
I will reset this file, but we need do cleancode later.